src/file_checker.py

```diff
@@ -29,17 +29,16 @@ def convert_sep(self, name, formats_):
                 name = name.replace('_', '-')
         return name
 
-    def convert_alphabet(self, name, formats_):
+    def convert_alphabet(self, name, format):
         try:
-            for f in formats_:
-                if f == 'upper_case':
-                    name = name.upper()
-                if f == 'lower_case':
-                    name = name.lower()
-                if f == 'camel_case':
-                    pass
-                if f == 'pascal_case':
-                    pass
+            if format == 'upper_case':
```

Comment:
I wouldn't use format as a name. It is a fairly common function that shouldn't cause any error here, but still.